docs/faq.rst

```diff
@@ -295,6 +295,15 @@ the :ref:`borg_recreate` command to rewrite all archives with a
 different ``--exclude`` pattern. See the examples in the
 :ref:`borg_recreate` manpage for more information.
 
+Can I safely change the compression level or algorithms?
+--------------------------------------------------------
+
+The compression level and algorithm don't affect deduplication. File
+hashes are calculated *before* compression. New compression settings
+will only be applied to new data, not existing archives. So it's safe
+to change them.
```

Comment:
Maybe we rather should refer to "chunks" here, as that is being processed (not: file, archive).

So:
- "Chunk ID hashes"
- "new chunks"
- "existing chunks"

Also we could mention that borg recreate can be used to recompress.